audio.go

```diff
@@ -1,7 +1,7 @@
 package vz
 
 /*
-#cgo darwin CFLAGS: -x objective-c -fno-objc-arc
+#cgo darwin CFLAGS: -x objective-c -fno-objc-arc -mmacosx-version-min=10.13
```

Comment:
Please remove this flag from each file
```suggestion
#cgo darwin CFLAGS: -x objective-c -fno-objc-arc
```

Reply:
Is it fine if it's `-mmacosx-version-min=10.13`? Or do you prefer if I set it to `CGO_LDFLAGS` in the Makefile?

Reply:
`-mmacosx-version-min=11.0` is needed in order to generate binaries which will build on older macOS versions. If it's not added to the go files, I think users of Code-Hex/vz will have to add it themselves to their build flags. Do you prefer to leave this choice to users of the module?

Reply:
Yes. Because I referred about the version should use above than 11.x in readme. So I don't think this user is used in older version